cinema/tests/test_movie_and_movie_session.py

```diff
@@ -0,0 +1,246 @@
+
+        self.assertEqual(
+            movie_serializer_Illia.data["actors"],
+            movie_responce_Illia.data[0]["actors"],
+        )
+        self.assertEqual(
+            movie_serializer_Jonh.data["actors"], movie_responce_Jonh.data[0]["actors"]
+        )
+
+        self.assertNotEqual(
+            movie_serializer_Jonh.data["actors"], movie_responce_Illia.data[0]["actors"]
+        )
+
+    def test_filter_movie_by_genres(self):
+        movie_response_horror = self.client.get(
+            MOVIE_LIST_URL, {"genres": f"{self.genre_horror.id}"}
+        )
+        movie_response_drama = self.client.get(
+            MOVIE_LIST_URL, {"genres": f"{self.genre_drama.id}"}
+        )
+
+        movie_serializer_horror = MovieListSerializer(self.movie_apalon, many=False)
+        movie_serializer_drama = MovieListSerializer(self.movie_vinston, many=False)
+
+        self.assertEqual(
+            movie_serializer_horror.data["genres"],
+            movie_response_horror.data[0]["genres"],
+        )
+        self.assertEqual(
+            movie_serializer_drama.data["genres"],
+            movie_response_drama.data[0]["genres"],
+        )
+        self.assertNotEqual(
+            movie_serializer_drama.data["genres"],
+            movie_response_horror.data[0]["genres"],
+        )
+
+    def test_filter_movie_session_by_date(self):
+        movie_session_response_2023 = self.client.get(
+            MOVIE_SESSION_LIST_URL,
+            {
+                "date": f"{self.movie_session_2023.show_time.year}-"
+                f"{self.movie_session_2023.show_time.month}-"
+                f"{self.movie_session_2023.show_time.day}"
+            },
+        )
+
+        movie_session_response_2024 = self.client.get(
+            MOVIE_SESSION_LIST_URL,
+            {
+                "date": f"{self.movie_session_2024.show_time.year}-"
+                f"{self.movie_session_2024.show_time.month}-"
+                f"{self.movie_session_2024.show_time.day}"
+            },
+        )
+
+        movie_session_serializer_2023 = MovieSessionListSerializer(
+            self.movie_session_2023, many=False
+        )
+
+        movie_session_serializer_2024 = MovieSessionListSerializer(
+            self.movie_session_2024, many=False
+        )
+
+        self.assertEqual(
+            movie_session_response_2023.data[0]["show_time"],
+            movie_session_serializer_2023.data["show_time"],
+        )
+
+        self.assertEqual(
+            movie_session_response_2024.data[0]["show_time"],
+            movie_session_serializer_2024.data["show_time"],
+        )
+
+        self.assertNotEqual(
+            movie_session_response_2024.data[0]["show_time"],
+            movie_session_serializer_2023.data["show_time"],
+        )
+
+    def test_filter_movie_session_by_movie_title(self):
+        movie_session_response_vinston = self.client.get(
+            MOVIE_SESSION_LIST_URL, {"movie": f"{self.movie_session_2023.movie.id}"}
```

Comment:
The same applies here. Add a test case for a movie that does not exist in the database.